src/extension/features/accounts/last-reconciled-date/index.js

```diff
@@ -73,13 +110,20 @@ export class LastReconciledDate extends Feature {
   };
 
   /**
-   * Helper method to show and hide the reconcile date container
+   * Helper methods to show and hide the reconcile containers
    * @param {Boolean} visible True to show the container, false to hide
    */
-  _setFeatureVisibility = visible => {
+  _setDateLastFeatureVisibility = visible => {
```

Comment:
Rather than duplicate this function for both days since reconciled and last reconciled date, lets just add a parameter for the feature container selector.

Something along the lines of
```
_setFeatureVisibility = (feature_selector, visible) => {
    ...
}
```

Reply:
thanks. I am relatively new to javascript so this might take some learning on my part to implement passing the feature container as a parameter.

Reply:
No worries (: We should be able to make this into a function

```
  _setFeatureVisibility = (feature_selector, visible) => {
    let featureContainer = $(feature_selector);
    if (featureContainer && featureContainer.length) {
      featureContainer.toggle(visible);
    }
  };
```

and then to call it we should just pass in the appropriate selector.
```
this._setFeatureVisibility('.tk-accounts-header-days-since-reconciled', true)
```

or

```
this._setFeatureVisibility('.tk-accounts-header-last-reconciled', true)
```

That way we can get rid of the code duplication.

Reply:
done.

Reply:
done.